zcash_test_vectors/unified_address.py

```diff
@@ -39,7 +39,11 @@ def main():
 
     test_vectors = []
     for account in range(0, 20):
+        # Each set of sequential diversified addresses should have the same set
+        # of typecodes, to simplify use in tests.
         has_t_addr = rand.bool()
+        # use p2pkh 3/4 of the time
+        is_p2pkh = rand.bool() or rand.bool()
```

Comment:
`or` is shortcutting, which might make it more difficult to refactor this code in a way that preserves existing test vector entries that should be unaffected.
```suggestion
        is_p2pkh = any([rand.bool(), rand.bool()])
```